addons/rose/app/styles/rose/variables/color/_css.scss

```diff
@@ -76,6 +76,8 @@
   --ui-gray-starker-3: #{color(ui-gray, 800)};
   --ui-gray-starker-4: #{color(ui-gray, 900)};
   --ui-header-subtler: #{color(ui-cool-gray, 700)};
+
+  color-scheme: light;
```

Comment:
added this so that `color-scheme: dark` gets overridden properly when light mode mixin is used.